ProductivitySuite/tests/test_productivity_suite_on_xeon.sh

```diff
@@ -0,0 +1,403 @@
+
+    docker images
+
+}
+
+function start_services() {
+    cd $WORKPATH/docker/xeon
+
+    export EMBEDDING_MODEL_ID="BAAI/bge-base-en-v1.5"
+    export RERANK_MODEL_ID="BAAI/bge-reranker-base"
+    export LLM_MODEL_ID="Intel/neural-chat-7b-v3-3"
+    export LLM_MODEL_ID_CODEGEN="meta-llama/CodeLlama-7b-hf"
+    export TEI_EMBEDDING_ENDPOINT="http://${ip_address}:6006"
+    export TEI_RERANKING_ENDPOINT="http://${ip_address}:8808"
+    export TGI_LLM_ENDPOINT="http://${ip_address}:9009"
+    export REDIS_URL="redis://${ip_address}:6379"
+    export REDIS_HOST=${ip_address}
+    export INDEX_NAME="rag-redis"
+    export HUGGINGFACEHUB_API_TOKEN=${HUGGINGFACEHUB_API_TOKEN}
+    export MEGA_SERVICE_HOST_IP=${ip_address}
+    export EMBEDDING_SERVICE_HOST_IP=${ip_address}
+    export RETRIEVER_SERVICE_HOST_IP=${ip_address}
+    export RERANK_SERVICE_HOST_IP=${ip_address}
+    export LLM_SERVICE_HOST_IP=${ip_address}
+    export LLM_SERVICE_HOST_IP_DOCSUM=${ip_address}
+    export LLM_SERVICE_HOST_IP_FAQGEN=${ip_address}
+    export LLM_SERVICE_HOST_IP_CODEGEN=${ip_address}
+    export LLM_SERVICE_HOST_IP_CHATQNA=${ip_address}
+    export TGI_LLM_ENDPOINT_CHATQNA="http://${ip_address}:9009"
+    export TGI_LLM_ENDPOINT_CODEGEN="http://${ip_address}:8028"
+    export TGI_LLM_ENDPOINT_FAQGEN="http://${ip_address}:9009"
+    export TGI_LLM_ENDPOINT_DOCSUM="http://${ip_address}:9009"
+    export BACKEND_SERVICE_ENDPOINT_CHATQNA="http://${host_ip}:8888/v1/chatqna"
+    export BACKEND_SERVICE_ENDPOINT_FAQGEN="http://${host_ip}:8889/v1/faqgen"
+    export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:6009/v1/dataprep/delete_file"
+    export BACKEND_SERVICE_ENDPOINT_CODEGEN="http://${host_ip}:7778/v1/codegen"
+    export BACKEND_SERVICE_ENDPOINT_DOCSUM="http://${host_ip}:8890/v1/docsum"
+    export DATAPREP_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/dataprep"
+    export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:6008/v1/dataprep/get_file"
+    export CHAT_HISTORY_CREATE_ENDPOINT="http://${host_ip}:6012/v1/chathistory/create"
+    export CHAT_HISTORY_CREATE_ENDPOINT="http://${host_ip}:6012/v1/chathistory/create"
+    export CHAT_HISTORY_DELETE_ENDPOINT="http://${host_ip}:6012/v1/chathistory/delete"
+    export CHAT_HISTORY_GET_ENDPOINT="http://${host_ip}:6012/v1/chathistory/get"
+    export PROMPT_SERVICE_GET_ENDPOINT="http://${host_ip}:6015/v1/prompt/get"
+    export PROMPT_SERVICE_CREATE_ENDPOINT="http://${host_ip}:6015/v1/prompt/create"
+    export KEYCLOAK_SERVICE_ENDPOINT="http://${host_ip}:8080"
+    export MONGO_HOST=${ip_address}
+    export MONGO_PORT=27017
+    export DB_NAME="opea"
+    export COLLECTION_NAME="Conversations"
+    export LLM_SERVICE_HOST_PORT_FAQGEN=9002
+    export LLM_SERVICE_HOST_PORT_CODEGEN=9001
+    export LLM_SERVICE_HOST_PORT_DOCSUM=9003
+    export PROMPT_COLLECTION_NAME="prompt"
+
+    if [[ "$IMAGE_REPO" != "" ]]; then
```

Comment:
Please update the `compose.yaml` like this one https://github.com/opea-project/GenAIExamples/blob/main/SearchQnA/docker/xeon/compose.yaml, so we can use env to pass value instead of sed operate.

Reply:
${REGISTRY:-opea}/web-retriever-chroma:${TAG:-latest}